src/test/java/org/jabref/logic/crawler/StudyCatalogToFetcherConverterTest.java

```diff
@@ -42,7 +42,6 @@ void setUpMocks() {
         when(saveConfiguration.getSaveOrder()).thenReturn(SaveOrder.getDefaultSaveOrder());
         when(preferencesService.getBibEntryPreferences().getKeywordSeparator()).thenReturn(',');
         when(preferencesService.getImporterPreferences().getApiKeys()).thenReturn(FXCollections.emptyObservableSet());
-
         entryTypesManager = new BibEntryTypesManager();
```

Comment:
Why this change? Not needed, is it?

Reply:
It is needed because of getCatalogs method